app/views/admin/legislation/questions/index.html.erb

```diff
@@ -32,6 +33,7 @@
             <td>
               <%= link_to question.title, edit_admin_legislation_process_question_path(@process, question) %>
             </td>
+            <td><%= sanitize_and_auto_link(question.description) %></td>
```

Comment:
The only place that comes to mind where we show CKEditor descriptions in admin tables are poll question answers, where we use `<%= wysiwyg(answer.description) %>` to display them :thinking:.

So I'd suggest either using `wysiwyg`, removing the description, or changing the table in poll question answers. Personally I'd vote to remove it because descriptions make tables harder to read when they're longer than one or two sentences :thinking:.